docs/administrator-handbook.md

```diff
@@ -94,6 +94,15 @@ that there are not other pending changes:
 ```
 # rpm-ostree install -yA <pkg>
 ```
+### Override rpm files
+
+The install --force-replacefiles feature introduces the ability to override files 
+from one RPM package with files from another RPM package.
```

Comment:
This says what it does, which is appropriate for usage text.  This section of the doc is more of a "guide", and it'd make sense to say *why* one might want this.  How about something like:

> Normally, RPM does not allow one package to overwrite files from another.  But it can make sense to relax this restriction in some cases; for example, where one just wants to overwrite one kernel module without rebuilding the whole kernel package.  The `--force-replacefiles` option allows this.

Reply:
Makes sense, ill add that in now thanks for the review!